backend/src/main/java/com/social/dailylink/service/PostServiceImpl.java

```diff
@@ -40,11 +38,12 @@ public Post create(Post entity) {
     }
 
     @Override
-    public Post save(Post entity) {
-        if (((PostRepository) repository).findByTitle(entity.getTitle()).isPresent()) {
-            throw new EntityAlreadyExistsException("Post " + entity.getTitle() + " already exists");
-        }
+    public List<Post> findAll() {
```

Comment:
You only have to override the Methods you want to make an own implementation of. The others are automatically "generated".